platform/glfw/glfw_view.cpp

```diff
@@ -379,12 +380,36 @@ void GLFWView::addRandomShapeAnnotations(int count) {
     }
 }
 
+void GLFWView::addRandomAnimatedAnnotations() {
```

Comment:
Love this functionality, but what's "random" about it?

Reply:
nothing, renaming